transform/transformers/ImageTransformer.py

```diff
@@ -1,43 +1,69 @@
-import os
-import glob
-import subprocess
-import zipfile
+#!/usr/bin/env python
+#   coding: UTF-8
+
+import argparse
 import datetime
 import dateutil.parser
```

Comment:
Name conflict with the module level parser function.
dateutil.parser is only called once so no performance impact wrt dictionary lookups, etc.